server/index.js

```diff
@@ -91,10 +94,15 @@ app.get('/api/:userId/favorites', (req, res, next) => {
       `;
   return db.query(sql)
     .then(result => {
-      res.status(201).json(result.rows);
+      res.json(result.rows);
     })
-    .catch(err => next(err));
-
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'an unexpected error occurred'
+      })
```

Comment:
I see you added a bunch of these but they weren't in the change request. Please try to do less work and make it count.